Sources/Valkey/ValkeyClient.swift

```diff
@@ -24,11 +24,9 @@ import _ValkeyConnectionPool
 import ServiceLifecycle
 #endif
 
-/// Valkey client
+/// A client that connects to a single Valkey server.
```

Comment:
This will not be correct in the near future, as ValkeyClient will support reading from replicas.